frontend/src/app/projects/models/model-source/choose-source.component.html

```diff
@@ -2,48 +2,65 @@
  ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
  ~ SPDX-License-Identifier: Apache-2.0
  -->
+<!-- New -->
 
-<div *ngIf="(projectService.project$ | async) && (modelService.model$ | async)">
-  <h2 class="title text-center">Git sources</h2>
-  <div class="flex flex-wrap justify-center">
-    <a (click)="modelSourceSelection.emit('git-add')">
-      <mat-card class="option-card">
-        <p>Use existing repository</p>
-        <app-mat-icon size="50px">link</app-mat-icon>
-      </mat-card>
-    </a>
-  </div>
+<div class="flex justify-center">
+  <mat-card class="mat-card-form" *ngIf="projectService.project$ | async">
+    <div
+      *ngIf="(projectService.project$ | async) && (modelService.model$ | async)"
+    >
+      <div
```

Comment:
Git sources should also be available for non-admins and Git sources don't require the T4C integration.

Reply:
Yes :)